modules/creating-a-project-workbench.adoc

```diff
@@ -14,51 +14,64 @@ endif::[]
 ifdef::upstream[]
 * If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
 endif::[]
-* You have created a data science project that you can add a workbench to.
+* You created a project. 
+ifndef::upstream[]
+* You created a Simple Storage Service (S3) account outside of {productname-long}. 
+* To create data connections to your existing S3-compatible storage buckets, you need the following credential information for the storage buckets: 	
```

Comment:
If this is optional, could make the wording a bit more parallel to the other prereqs:

'If you need to create data connections to your existing S3-compatible storage buckets, you have the following credential information:'